test/stripe/core_resources/payout_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule Stripe.PayoutTest do
+  use Stripe.StripeCase, async: true
+
+  describe "create/2" do
+    test "creates a card for a customer" do
+      assert {:ok, %Stripe.Payout{}} =
+               Stripe.Payout.create(%{amount: 100, currency: "USD", source_type: "card"})
```

Comment:
I would rewrite this to take the params and assign them to an intermediate variable:

```
params = %{amount: 100, currency: "USD", source_type: "card"}
assert {:ok, %Stripe.Payout{}} = Stripe.Payout.create(params)
```